### PR TITLE
Fix URLError when installing napp from remote

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: kytos/cli/commands/napps/api.py [hunk 1]
```diff
@@ -159,6 +159,8 @@ def install_napp(cls, mgr):
                     log.error('  NApp not found.')
                 else:
                     log.error('  NApps Server error: %s', e)
+            except URLError as e:
+                log.error('  NApps Server error: '+str(e.reason))
```
Comment (diraol) [inline kytos/cli/commands/napps/api.py]: @macartur, please, fix this logging statement. This syntax (`+`) is not in compliance with our coding style guidelines.

Comment (cemsbr) [inline kytos/cli/commands/napps/api.py]: Note: check line 161.

Comment (macartur) [inline kytos/cli/commands/napps/api.py]: Done.

**Comment by macartur** [comment]
Done.

**Pull request merged**